src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,124 @@
+.footer {
+  position: relative;
+  background-color: $c-background-grey;
+  color: $c-light-gray;
+
+  &__section {
+    padding: 80px 0 40px;
```

Comment:
paddings is not the same for all resolutions
for desktop they are 160px 0 60px
for tablet -  120px 0 35px